README.md

```diff
@@ -36,13 +36,27 @@ For platform-specific details on installing go, see: https://go.dev/doc/install
 
 # Quick Setup
 
+## 0. Build the App
+
+If you haven't already, the first step is to clone and build the app locally:
+
+```bash
+$ git clone https://github.com/Linaro/lite_bootstrap_server.git
+$ cd lite_bootstrap_server
+$ go build -o liteboot
+```
+
+> The `-o liteboot` is optional, but will cause the compiled app to have the
```

Comment:
The other option would be to put main inside a subdirectory called the name we want for liteboot. That would probably be best as a new pr, though.